basket/BABY_LOUIE/src/js/global.js

```diff
@@ -6,15 +6,16 @@
 import jQuery from 'jquery';
 
 (function ($, Drupal) {
-  "use strict";
+  // 'use strict';
```

Comment:
Can this comment be removed?

Reply:
Drupal standards require `use strict` I'll uncomment and test.